src/main/java/com/upskill/rentcars/service/CarService.java

```diff
@@ -92,7 +96,9 @@ public boolean isField(int field) {
     }
 
     private String setServerImageUrl(){
-        String[] imageNames = {"car1.png", "car2.png", "car3.png"};
-        return ServletUriComponentsBuilder.fromCurrentContextPath().path("/cars/image" + imageNames[2]).toUriString();
+        Random random = new Random();
+        int pictureNumber = random.nextInt(10);
+        return ServletUriComponentsBuilder.fromCurrentContextPath().path(
+                "/cars/images/car/" + pictureNumber).toUriString();
```

Comment:
Your url must have a template like this "..car/car-1.png".
Now it is "..car/1".

Reply:
And add cars images.